blivet/devicelibs/stratis.py

```diff
@@ -0,0 +1,96 @@
+#
+# stratis.py
+# stratis functions
+#
+# Copyright (C) 2020  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Author(s): Vojtech Trefny <vtrefny@redhat.com>
+#
+
+import gi
+gi.require_version("GLib", "2.0")
+
+from gi.repository import GLib
+
+from ..errors import StratisError
+from ..size import Size
+from ..static_data import stratis_info
+from .. import safe_dbus
+
+
+STRATIS_SERVICE = "org.storage.stratis2"
+STRATIS_PATH = "/org/storage/stratis2"
+STRATIS_POOL_INTF = STRATIS_SERVICE + ".pool"
+STRATIS_FILESYSTEM_INTF = STRATIS_SERVICE + ".filesystem"
+STRATIS_BLOCKDEV_INTF = STRATIS_SERVICE + ".blockdev"
+STRATIS_PROPS_INTF = STRATIS_SERVICE + ".FetchProperties"
+STRATIS_MANAGER_INTF = STRATIS_SERVICE + ".Manager"
```

Comment:
A few notes about the interface design that we have. Because we try to be as backwards compatible as possible, these interface names should still be available in Stratis v2.x, but just a note that these are the oldest interfaces available. These will probably need to be updated as new Fedora releases include later versions of stratisd.

Reply:
jbaublitz is correct. You'll generally want the most recent revision for the version of stratisd that is available.

Reply:
I've used these interfaces mostly because I didn't need any properties from the newer revisions, but I'll try to use the newest interfaces in the future.